GooglePlacesAutocomplete.d.ts

```diff
@@ -401,19 +401,20 @@ interface GooglePlacesAutocompleteProps {
   predefinedPlaces?: Place[];
   predefinedPlacesAlwaysVisible?: boolean;
   preProcess?: (text: string) => string;
-  query: Query;
+  query: Query | Object;
   renderDescription?: (description: DescriptionRow) => string;
-  renderHeaderComponent?: () => React.ComponentType<{}>;
-  renderLeftButton?: () => React.ComponentType<{}>;
-  renderRightButton?: () => React.ComponentType<{}>;
-  renderRow?: (data: GooglePlaceData) => React.ComponentType<{}>;
+  renderHeaderComponent?: () => JSX.Element | React.ComponentType<{}>;
+  renderLeftButton?: () => JSX.Element | React.ComponentType<{}>;
+  renderRightButton?: () => JSX.Element | React.ComponentType<{}>;
+  renderRow?: (data: GooglePlaceData) => JSX.Element | React.ComponentType<{}>;
+  
   // sets the request URL to something other than the google api.  Helpful if you want web support or to use your own api.
   requestUrl?: RequestUrl;
-  styles?: Partial<Styles>;
+  styles?: Partial<Styles> | Object;
```

Comment:
Why is this necessary?

Reply:
TS complained when passing an object into the styles prop if it was not inline.

Reply:
This (and all the other "| Object" fields) allows any object to be passed as style, effectively breaking type checking. @rgordon95 I have had no issue passing in a non-inlined styles prop, and can't see a reason why doing so might have cause a typescript error unless the object passed was not of the correct type.